src/savedata.c

```diff
@@ -845,585 +838,558 @@ int SaveTableEntry_BodySize (int saveTableID)
     return size;
 }
 
-static void sub_020251DC (SavePageInfo * param0)
+static void SavePageInfo_Init (SavePageInfo *pageInfo)
 {
-    const SaveTableEntry * v0 = gSaveTable;
-    int v1;
-    int v2 = 0;
+    const SaveTableEntry *saveTable = gSaveTable;
```

Comment:
Odd choice. Do the accesses below not match unless this is a stack-variable?

Reply:
Nope, it only matches as a stack variable.

Reply:
Lame. Just GameFreak things.